#[
IOError: IO operation on closed file.
Traceback:
    File "[builtin]", from file::readline
    File "read_from_closed.lly", from __main__ at line 9
]#

stdin.close()
stdin.readline()
